### DISCUSSION TITLE
How does the Reference Architecture handle IPv6?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#119] [Help] [answered]
How does the Reference Architecture handle IPv6? Can we use IPv6 with our Reference Architecture?

**Comment by rhoboat** [comment, marked as answer]
Please follow the issue we are tracking to add support for IPv6, [here](https://github.com/gruntwork-io/terraform-aws-vpc/issues/247). Once we have it at the module level, we can propagate it to the service layer and then into the Ref Arch.
